src/Marten/Linq/Parsing/EnumerableContains.cs

```diff
@@ -19,10 +19,10 @@ public bool Matches(MethodCallExpression expression)
                expression.Arguments.Single().IsValueExpression();
     }
 
-    public ISqlFragment Parse(IFieldMapping mapping, ISerializer serializer, MethodCallExpression expression)
+    public ISqlFragment Parse(IFieldMapping mapping, IReadOnlyStoreOptions options, MethodCallExpression expression)
```

Comment:
Nice, @mysticmind, good that you removed also the obsolete serialiser param :+1: